<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogForSettings</class>
 <widget class="QDialog" name="DialogForSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>检测区域参数显示</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>方框1数据：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineText_box1"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>方框2数据：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineText_box2"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="4">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_detect">
       <property name="text">
        <string>监控区域设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_save">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_advance">
       <property name="text">
        <string>&amp;Advance</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>远程服务器设置</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>服务器IP地址：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="remoteIP"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>服务器传输数据端口：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="remotePort_data"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>服务器白名单端口：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="remotePort_whiteList"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_wifi">
     <property name="title">
      <string>Wi-Fi设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>探针MAC地址：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="MacAddress"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>探针端口：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="wifiPortLineText"/>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>时间间隔：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_interval"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>探针阈值：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="wifiThdLineText"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>-70</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_wifiThd">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>-20</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_Path">
     <property name="title">
      <string>路径设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>部分分割存储路径：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_segment"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_segment">
          <property name="text">
           <string>选择文件夹</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>本地身份证照片存取路径：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_idcard"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_idcard">
          <property name="text">
           <string>选择文件夹</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_otherPath">
     <property name="title">
      <string>其他路径设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>ini文件路径：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_iniPath"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_iniChoose">
          <property name="text">
           <string>选择文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_iniSave">
          <property name="text">
           <string>保存路径</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_advance</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_otherPath</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
